nipype/pipeline/engine/base.py

```diff
@@ -18,17 +18,15 @@
                         absolute_import)
 from builtins import object
 
-from future import standard_library
-standard_library.install_aliases()
-
 from copy import deepcopy
 import re
 import numpy as np
-from ... import logging
+from future import standard_library
+
 from ...interfaces.base import DynamicTraitedSpec
 from ...utils.filemanip import loadpkl, savepkl
 
-logger = logging.getLogger('workflow')
+standard_library.install_aliases()
```

Comment:
The aliases would modify the behavior of imports, so I guess that is the reason. Anyways, I moved it because flake8 suggested so. I can roll back at any moment

Reply:
@mgxd @satra Are you familiar enough with `future.standard_library.install_aliases` to know whether there's a semantic change being made by reordering here?

You can also add a `# noqa` comment to pacify flake8, if you do return it.

Reply:
some comments on the order are here: https://pypi.python.org/pypi/future

All examples [here](http://python-future.org/imports.html) have it at the very beginning